helpers/check_acme_routes.sh

```diff
@@ -148,37 +172,46 @@ function check_routes() {
 			fi
 
 			echo "$DNS_ERROR"
```

Comment:
is this echo supposed to be here?

Reply:
Yes. I added a comment # Print the error on stdout